client/utils/embedded-components/utils.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * External dependencies
+ */
+import apiFetch from '@wordpress/api-fetch';
+
+/**
+ * Internal dependencies
+ */
+import { NAMESPACE } from 'data/constants';
+import { addQueryArgs } from '@wordpress/url';
```

Comment:
move this to external dependencies

Reply:
Good catch! Moved it to the right place.